app/functions/getGMBidsSearchFilterQueries.js

```diff
@@ -0,0 +1,150 @@
+const { Op } = require("sequelize");
+module.exports = (query) => {
+  /*  getSearchQuery is a function created to get the where object 
+        with the filter by LIKE clause for email/mobile/name search
+        and only including contacts of a particular user uuid 
+    
+    */
+  let searchItem = query.search;
+
+  console.log("gm-products searchItems:", searchItem);
+
+  const productNames = query?.product_names
+    ?.split(",")
+    ?.map((productName) => productName);
+  const brands = query?.brands?.split(",")?.map((brand) => brand);
+  const cities = query?.cities?.split(",")?.map((city) => city);
```

Comment:
@maheshmuttinti Why are we searching for cities, and country, more importantly, are we adding these details into products anywhere.

Reply:
The user would add country and city while he adding products. That's why I have added the country and cities searching. Can you tell me if this is necessary or not?

Reply:
I removed this function since it was wrong implementation. @rajeshn95